crypto-jni/src/lib.rs

```diff
@@ -408,3 +467,199 @@ pub extern "system" fn Java_com_concordium_sdk_crypto_CryptoJniNative_generateCo
 
     return CryptoJniResult::Ok(payload).to_jstring(&env);
 }
+
+type SerializeParamResult = CryptoJniResult<String>;
+
+#[no_mangle]
+#[allow(non_snake_case)]
+/**
+ * The JNI wrapper for serializing receive parameters.
+ * Constructs a SerializeParamResult containg the hex encoded serialized parameter. 
```

Comment:
what if it fails? Can you add a few more sentences to this. 

Also perhaps some preconditions of the inputted parameters?